_summaries/loshchilov2017decoupled_2.md

```diff
@@ -3,17 +3,46 @@ layout: summary
 title: Summary
 paper: {{paper_tag}}
```

Comment:
Please replace `{{page_tag}}` with `loshchilov2016sgdr `. Also the file to edit is `loshchilov2016sgdr_2.md` instead of `loshchilov2017decoupled_2.md`, otherwise we will probably encounter conflicts when merging.

Reply:
Done!